src/main/java/nl/novi/techiteasycontroller/Controller/TelevisionController.java

```diff
@@ -0,0 +1,57 @@
+package nl.novi.techiteasycontroller.Controller;
+
+import nl.novi.techiteasycontroller.Exceptions.RecordNotFoundException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+
+@RestController
+public class TelevisionController {
+
+
+    ArrayList<String> televisionDatabase = new ArrayList<String>();
```

Comment:
In het kader van LSP (Polymorphisme) is het beter om te zeggen `List<String> xx = new ArrayList<>();`